pkg/kubelet/kubelet.go

```diff
@@ -775,12 +778,14 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 		}
 	}
 
+	kubeDeps.EventBroadcaster = toolsevents.NewBroadcaster(&toolsevents.EventSinkImpl{Interface: klet.kubeClient.EventsV1beta1().Events("")})
```

Comment:
`kubeDeps.EventBroadcaster` needs to call `eventBroadcaster.StartRecordingToSink(stopCh)` in `run` method.

Reply:
There doesn't seem to be stopCh in kubelet.go

Reply:
`kubelet` starts in `server.go`, and there is a `run` method with `stopCh`.